.github/workflows/test.yml

```diff
@@ -14,15 +14,13 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: 14.x
-      - name: Install Dependencies
-        run: npm install cddl parse5
-      - name: Extract CDDL content from spec into files
-        run: ./scripts/cddl/generate.js
-      - name: Print remote.cddl
-        run: cat remote.cddl
-      - name: Print local.cddl
-        run: cat local.cddl
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
       - name: Validate CDDL files
-        run: |
-          npx cddl validate local.cddl
-          npx cddl validate remote.cddl
+        run: ./scripts/test.sh --install
+      - name: Archive CDDL files
+        uses: actions/upload-artifact@v3
```

Comment:
👍🏻 for uploading the generated files as artifacts. Now it could be used by consumers to grab the validated files pretty easily.